lang/cs/Org.Apache.REEF.Wake/Remote/Impl/TransportServer.cs

```diff
@@ -34,9 +35,10 @@ public class TransportServer<T> : IDisposable
     {
         private static readonly Logger LOGGER = Logger.GetLogger(typeof(TransportServer<>));
 
-        private readonly TcpListener _listener;
+        private TcpListener _listener;
```

Comment:
If `FindAPortAndStartListener` would return a `TcpListener`, we could still keep the field immutable, making it easier to reason about the thread-safety of the class.